src/config.rs

```diff
@@ -575,6 +576,15 @@ fn load_global_config_file() -> Result<(PathBuf, Ini)> {
     }
 }
 
+fn get_failed_cli_config_load_msg(file_desc: &str) -> String {
+    let msg = format!("Failed to load {file_desc}.");
+    #[cfg(target_os = "macos")]
+    if xcode::launched_from_xcode() {
+        return msg + " Hint: Please ensure that ${SRCROOT}/.sentryclirc is added to the Input Files of this Xcode Build Phases script.";
```

Comment:
Also nitpick: I slightly preferred your previous implementation with `push_str` here.

Reply:
@szokeasaurusrex Clippy would give an error for having msg as mutable